### PR TITLE
Win lose

### PR DESCRIPTION
**Pull request by GDWR**
Added in LoS to lose game and win tile.
Need level transition for win and a UI for lose

closing #53 
closing #20 
closing #52

### CODE REVIEW

File: game/__main__.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 from typing import Optional
 from enum import Enum
 
+from game.entity.exit import Exit
```
Comment (SavagePastaMan) [inline game/__main__.py:4]: use a relative import here?

File: game/__main__.py [hunk 2]
```diff
@@ -54,6 +54,14 @@ def setup(self):
         self.set_viewport_on_player()
         self._draw()
 
+    def win_level(self):
+        # TODO: Transition to next level
+        print("You are built different")
+
+    def lose_level(self):
+        # TODO: Lose Ui
```
Comment (SavagePastaMan) [inline game/__main__.py:62]: i think we agreed to not have a loseview, it's easier to just reset the level

**Review by SavagePastaMan — CHANGES_REQUESTED** [review]
💯

File: game/__main__.py (hunk 2 above)
Comment (GDWR) [reply game/__main__.py:62]: Fair, we can just leave it as is

File: game/__main__.py (hunk 1 above)
Comment (GDWR) [reply game/__main__.py:4]: 🤢

**Comment by GDWR** [comment]
🥇 
tenor

**Pull request merged**